README.md

```diff
@@ -201,23 +187,21 @@ I would like to thank Microverse for their continued support. They've provided u
 
 ## ❓ FAQ (OPTIONAL) <a name="faq"></a>
 
-> Add at least 2 questions new developers would ask when they decide to use your project.
-
 - **Can I reuse this code?**
 
   - Yes sure, feel free.
 
 - **Are contributions welcome?**
 
-  - Yes please. Contact me at `djmulinde@gmail.com` and I will be happy to add you as a contributor.
+  - Yes please. Contact me at `mr.nejjari.abdelhafid@gmail.com` and I will be happy to add you as a contributor.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](./LICENSE.md) licensed.
 
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Kindly make sure the app solves the N+1 problem when fetching all Recipes for a user as requested in the project requirements. For more documentation, visits the [Solving the N+1 problem in Rails](https://medium.com/idopterlabs/solving-the-n-1-problem-in-rails-b5ea677e78e0)


- [ ] Kindly, Create a navigation menu that allows users to open all of the pages you created. 👇🏻
- [ ] Let's make sure to write Unit and Integration tests ✍🏻 

![image](https://user-images.githubusercontent.com/79058364/224495364-467ddf39-9062-44ff-9cd0-e0d0844df559.png)